src/citra_qt/debugger/graphics/graphics_surface.cpp

```diff
@@ -313,92 +313,95 @@ void GraphicsSurfaceWidget::Pick(int x, int y) {
         case Format::RG8: {
             auto value = Color::DecodeRG8(pixel) / 255.0f;
-            return QString("Red: %1, Green: %2")
+            return QStringLiteral("Red: %1, Green: %2")
                 .arg(QString::number(value.r(), 'f', 2))
                 .arg(QString::number(value.g(), 'f', 2));
         }
         case Format::I8:
-            return QString("Index: %1").arg(*pixel);
+            return QStringLiteral("Index: %1").arg(*pixel);
         case Format::A8:
-            return QString("Alpha: %1").arg(QString::number(*pixel / 255.0f, 'f', 2));
+            return QStringLiteral("Alpha: %1").arg(QString::number(*pixel / 255.0f, 'f', 2));
         case Format::IA4:
-            return QString("Index: %1, Alpha: %2").arg(*pixel & 0xF).arg((*pixel & 0xF0) >> 4);
+            return QStringLiteral("Index: %1, Alpha: %2")
+                .arg(*pixel & 0xF)
+                .arg((*pixel & 0xF0) >> 4);
         case Format::I4: {
             u8 i = (*pixel >> ((offset % 2) ? 4 : 0)) & 0xF;
-            return QString("Index: %1").arg(i);
+            return QStringLiteral("Index: %1").arg(i);
         }
         case Format::A4: {
             u8 a = (*pixel >> ((offset % 2) ? 4 : 0)) & 0xF;
-            return QString("Alpha: %1").arg(QString::number(a / 15.0f, 'f', 2));
+            return QStringLiteral("Alpha: %1").arg(QString::number(a / 15.0f, 'f', 2));
         }
         case Format::ETC1:
         case Format::ETC1A4:
             // TODO: Display block information or channel values?
-            return QString("Compressed data");
+            return QStringLiteral("Compressed data");
         case Format::D16: {
             auto value = Color::DecodeD16(pixel);
-            return QString("Depth: %1").arg(QString::number(value / (float)0xFFFF, 'f', 4));
+            return QStringLiteral("Depth: %1").arg(QString::number(value / (float)0xFFFF, 'f', 4));
         }
         case Format::D24: {
             auto value = Color::DecodeD24(pixel);
-            return QString("Depth: %1").arg(QString::number(value / (float)0xFFFFFF, 'f', 4));
+            return QStringLiteral("Depth: %1")
+                .arg(QString::number(value / (float)0xFFFFFF, 'f', 4));
         }
         case Format::D24X8:
         case Format::X24S8: {
             auto values = Color::DecodeD24S8(pixel);
-            return QString("Depth: %1, Stencil: %2")
+            return QStringLiteral("Depth: %1, Stencil: %2")
                 .arg(QString::number(values[0] / (float)0xFFFFFF, 'f', 4))
                 .arg(values[1]);
         }
         case Format::Unknown:
-            return QString("Unknown format");
+            return QStringLiteral("Unknown format");
         default:
-            return QString("Unhandled format");
+            return QStringLiteral("Unhandled format");
         }
-        return QString("");
+        return QString{};
     };
 
-    QString nibbles = "";
+    QString nibbles = QString{};
```

Comment:
```suggestion
    QString nibbles;
```

QStrings are empty by default.